salt/ext/importlib_metadata/__init__.py

```diff
@@ -41,15 +41,22 @@
     NullFinder,
     PyPy_repr,
     install,
-    Protocol,
+    # Protocol,
```

Comment:
removed?

Reply:
Yes, salt currently makes no use of typing, specially on `salt/ext` and typing on 3.5 also reduces performance.